### ISSUE TITLE
Code editor 2: Middle clicking/pressing F1 on a method inherited from a parent object opens bugged code tabs

### ISSUE THREAD

**Issue by KormexGit** [YoYoGames/GameMaker-Bugs#5310]
### Description
When calling a parent's method on a child object, middle clicking to jump to the definition of the method will open up a bugged code window for the parent. If the parent's code tab was already open, it will open a 2nd tab. This tab will be bugged in that edits can't be made in it, since any changes won't save. It's bugged even if it's the only tab open.

If you call the parent method inside a different method on the child, a similar issue will happen, but it won't jump to the definition at all and will instead open a 2nd bugged tab for the child object.

https://github.com/YoYoGames/GameMaker-Bugs/assets/100942509/2133e6d2-6280-4912-99c9-42917fa801ac

### Steps To Reproduce
On an object: 
```
parentMethod = function() {}
```
On an object that is a child of the above object:
```
childMethod = function() {
    parentMethod(); //Middle clicking here will open a bugged child object tab
}
parentMethod(); //Middle clicking here will open a bugged parent object tab
```

### Which version of GameMaker are you reporting this issue for?
IDE v2024.400.0.532 Runtime v2024.400.0.551

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22631.0

[73d88d55-d83b-49b1-8bbb-a1711dbf90eb](https://api.gamemaker.io/api/github/downloads/73d88d55-d83b-49b1-8bbb-a1711dbf90eb)

**Comment by yyalansavage** [comment]
Fixed; note this issue was only occurring for project closed and reopened in same IDE session

**Comment by sihammill** [comment]
Fix verified in IDE v2024.400.0.534  Runtime v2024.400.0.553
